discovery/marathon/marathon.go

```diff
@@ -188,23 +193,35 @@ type Task struct {
 	Ports []uint32 `json:"ports"`
 }
 
+// PortMappings describes in which port the process are binding inside the docker container
+type PortMappings struct {
+	Labels map[string]string `json:"labels"`
+}
+
 // DockerContainer describes a container which uses the docker runtime.
 type DockerContainer struct {
-	Image string `json:"image"`
+	Image        string         `json:"image"`
+	PortMappings []PortMappings `json:"portMappings"`
 }
 
 // Container describes the runtime an app in running in.
 type Container struct {
 	Docker DockerContainer `json:"docker"`
 }
 
+// PortDefinitions describes which load balancer port you should access to access the service
```

Comment:
Comments in Prometheus code bases should always be complete sentences, including the dot at the end.